Tests/js/views/actions/assets/ez-translateactionview-tests.js

```diff
@@ -0,0 +1,334 @@
+            });
+            Mock.expect(this.versionMock, {
+                method: 'getTranslationsList',
+                returns: this.translationsList
+            });
+            Mock.expect(this.locationMock, {
+                method: 'toJSON',
+                returns: {}
+            });
+            Mock.expect(this.contentMock, {
+                method: 'toJSON',
+                returns: {}
+            });
+
+            this.view = new Y.eZ.TranslateActionView({
+                container: '.container',
+                actionId: 'translate',
+                label: 'Translations',
+                disabled: false,
+                content: this.contentMock,
+                location: this.locationMock
+            });
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+            delete this.view;
+        },
+
+        'Should expand the view when receiving the translateAction event': function () {
+            this.view.render();
+            this.view.set('expanded', false);
+            this.view.fire('translateAction');
+
+            Assert.isTrue(this.view.get('expanded'), "The expanded state should be true");
+        },
+
+        'Should hide the view when receiving the translateAction event': function () {
+            this.view.render();
+            this.view.set('expanded', true);
+            this.view.fire('translateAction');
+
+            Assert.isFalse(this.view.get('expanded'), "The expanded state should be false");
+        },
+    });
+
+    renderTest = new Y.Test.Case({
+        name: 'eZ Translate Action View render test',
+
+        setUp: function () {
+            this.contentMock = new Mock();
+            this.locationMock = new Mock();
+            this.versionMock = new Mock();
+            this.translationsList = ['eng-GB', 'pol-PL', 'ger-DE', 'fre-FR'];
+
+            Mock.expect(this.contentMock, {
+                method: 'get',
+                args: ['currentVersion'],
+                returns: this.versionMock
+            });
+            Mock.expect(this.versionMock, {
+                method: 'getTranslationsList',
+                returns: this.translationsList
+            });
+            Mock.expect(this.locationMock, {
+                method: 'toJSON',
+                returns: {}
+            });
+            Mock.expect(this.contentMock, {
+                method: 'toJSON',
+                returns: {}
+            });
+
+            this.view = new Y.eZ.TranslateActionView({
+                container: '.container',
+                actionId: 'translate',
+                label: 'Translations',
+                disabled: false,
+                content: this.contentMock,
+                location: this.locationMock
+            });
+            this.view.render();
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+            delete this.view;
+        },
+
+        'Should render links': function () {
```

Comment:
this test is kind of useless because it actually tests the template of the view (and the .html uses a custom minimal one) so it should be removed. (The templates are/will be tested with the BDD tests).